packages/sdk/src/controllers/PageController.ts

```diff
@@ -61,4 +61,15 @@ export class PageController {
         const res = await this.#editorAPI;
         return res.setPageHeight(id, height).then((result) => getEditorResponseData<null>(result));
     };
+
+    /**
+     * This method changes the order of pages.
+     * @param order the index in the list to move to
+     * @param ids An array of all IDs you want to move to the given index
+     * @returns
+     */
+    reorder = async (order: number, ids: Id[]) => {
```

Comment:
https://chilipublishintranet.atlassian.net/browse/WRS-2075
According to my proposal (and aligned with same functionality on other places) it should be page.move (f.e. actions, characterstyles, ...)